app/assets/stylesheets/comments.scss

```diff
@@ -0,0 +1,39 @@
+/*
+  Place all the styles related to the matching controller here.
+  They will automatically be included in application.css.
+*/
+
+.comments-form {
+  margin-top: -25px;
+  margin-bottom: 70px;
+}
+
+.comments {
+  margin: 30px 0 70px 0;
```

Comment:
Properties should be ordered background, margin, padding<br>Shorthand form for property `margin` should be written more concisely as `30px 0 70px` instead of `30px 0 70px 0`